src/main/java/frc/robot/constants/IDConstants.java

```diff
@@ -19,7 +19,8 @@ public class IDConstants {
     public static final int FR_ABS = 10;
     public static final int BL_ABS = 11;
     public static final int BR_ABS = 2;
-
+    
+    public static final int OUTTAKE_MOTOR = 9;
```

Comment:
Maybe we should change this constant's name to ```SHOOTER_MOTOR``` in order to not get it confused with ```INTAKE_MOTOR```